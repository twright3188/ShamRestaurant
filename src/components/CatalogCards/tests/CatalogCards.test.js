import React from 'react';
import renderer from 'react-test-renderer';
import CatalogCardsItem from '../index';

it('catalog card renders correctly', () => {
    const tree = renderer.create(<CatalogCardsItem image='https://i.odealarose.com/12/images/produits/chloe-white-fall-bouquet-ode-a-la-rose-550x550-36857-1.jpg?_ga=2.193642541.1665858073.1572425823-1452047899.1572425823&_gac=1.188734938.1572426511.Cj0KCQjw6eTtBRDdARIsANZWjYaM6e1RrMtApYkqx8l0mvaOyCvPv4pEOu3HZNPca1zKIp2-tBHLj9UaArhrEALw_wcB'>test</CatalogCardsItem>).toJSON();
    expect(tree).toMatchSnapshot();
});