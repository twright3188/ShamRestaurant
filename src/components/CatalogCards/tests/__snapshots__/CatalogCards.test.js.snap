// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`catalog card renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "flex-start",
      "backgroundColor": "#fff",
      "borderRadius": 16,
      "elevation": 1,
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "center",
      "marginBottom": 16,
      "shadowColor": "rgba(128,128,128, 1)",
      "shadowOffset": Object {
        "height": 2,
        "width": 0,
      },
      "shadowOpacity": 0.2,
      "shadowRadius": 1,
      "width": "100%",
    }
  }
>
  <Image
    resizeMode="contain"
    source={
      Object {
        "uri": "https://i.odealarose.com/12/images/produits/chloe-white-fall-bouquet-ode-a-la-rose-550x550-36857-1.jpg?_ga=2.193642541.1665858073.1572425823-1452047899.1572425823&_gac=1.188734938.1572426511.Cj0KCQjw6eTtBRDdARIsANZWjYaM6e1RrMtApYkqx8l0mvaOyCvPv4pEOu3HZNPca1zKIp2-tBHLj9UaArhrEALw_wcB",
      }
    }
    style={
      Object {
        "borderTopLeftRadius": 16,
        "borderTopRightRadius": 16,
        "height": 345,
        "overflow": "hidden",
        "width": "100%",
      }
    }
  />
  <View
    style={
      Object {
        "maxWidth": "100%",
        "padding": 16,
      }
    }
  >
    test
  </View>
</View>
`;
